### PR TITLE
Make 3gp signature more lenient to match spec

### LINKED ISSUE #105: Support for 3gp
Hello, 

Could you add the support of 3gp files ? For information, 3gp files are identified by the string "ftyp3gp4" at the offset 3.

Best regards,
Seb

### PR DESCRIPTION
**Pull request by BrandonSmith**
This pull request adds a fixture for pull request (#124) @dominic-p provided that fixes #105.

### CODE REVIEW

File: test.js [hunk 1]
```diff
@@ -93,6 +93,7 @@ const types = [
 ];
 
 const names = {
+	'3gp': ['fixture', 'fixture2'],
```
Comment (karlhiramoto) [inline test.js]: should be consistent and not use quotes.

Comment (BrandonSmith) [reply test.js]: @karlhiramoto I'm open to suggestions. However, unquoting `'3gp'` would be a syntax error.

**Comment by sindresorhus** [comment]
Same comment as https://github.com/sindresorhus/file-type/pull/127#issuecomment-365172265

File: test.js (hunk 1 above)
Comment (sindresorhus) [reply test.js]: Quoted is fine here as it's needed.

**Comment by dominic-p** [comment]
The readme and package.json already list 3gp support. All this PR does is fix the signature. Unless I'm missing something...

**Comment by sindresorhus** [comment]
@dominic-p You're right. My mistake.

**Pull request merged**